vips/vip-converter/vip-converter-testnet/commands.ts

```diff
@@ -0,0 +1,168 @@
+  const validTokenIns = [BaseAssets[2], BaseAssets[3], BaseAssets[4], BaseAssets[5]];
+
+  if (validTokenIns.includes(tokenIn) && tokenOut === BaseAssets[0]) {
+    return [0, 2];
+  } else {
+    return [0, 1];
+  }
+}
+
+function generateAcceptOwnershipCommands(ConvertersArray: string[]): AcceptOwnership[] {
+  const acceptOwnershipCommandsArray: AcceptOwnership[] = [];
+
+  for (const converter of ConvertersArray) {
+    const config: AcceptOwnership = {
+      target: converter,
+      signature: "acceptOwnership()",
+      params: [],
+    };
+
+    acceptOwnershipCommandsArray.push(config);
+  }
+
+  return acceptOwnershipCommandsArray;
+}
+
+function generateAddTokenConverterCommands(ConvertersArray: string[]): AddTokenConverter[] {
+  const addTokenConverterCommandsArray: AddTokenConverter[] = [];
+
+  for (const converter of ConvertersArray) {
+    const config: AddTokenConverter = {
+      target: CONVERTER_NETWORK,
+      signature: "addTokenConverter(address)",
+      params: [converter],
+    };
+
+    addTokenConverterCommandsArray.push(config);
+  }
+  return addTokenConverterCommandsArray;
+}
+
+function generateAddConverterNetworkCommands(ConvertersArray: string[]): AddConverterNetwork[] {
+  const addConverterNetworkCommandsArray: AddConverterNetwork[] = [];
+
+  for (const converter of ConvertersArray) {
+    const config: AddConverterNetwork = {
+      target: converter,
+      signature: "setConverterNetwork(address)",
+      params: [CONVERTER_NETWORK],
+    };
+
+    addConverterNetworkCommandsArray.push(config);
+  }
+  return addConverterNetworkCommandsArray;
+}
+
+function generateCallPermissionCommands(ConvertersArray: string[]): CallPermissionConverters[] {
+  const callPermissionCommandsArray: CallPermissionConverters[] = [];
+
+  for (const converter of ConvertersArray) {
+    const config1: CallPermissionConverters = {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [converter, "setConversionConfig(address,address,ConversionConfig)", NORMAL_TIMELOCK],
+    };
+    const config2: CallPermissionConverters = {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [converter, "pauseConversion()", NORMAL_TIMELOCK],
+    };
+    const config3: CallPermissionConverters = {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [converter, "resumeConversion()", NORMAL_TIMELOCK],
+    };
+    const config4: CallPermissionConverters = {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [converter, "setMinAmountToConvert(uint256)", NORMAL_TIMELOCK],
+    };
+
+    callPermissionCommandsArray.push(config1);
+    callPermissionCommandsArray.push(config2);
+    callPermissionCommandsArray.push(config3);
+    callPermissionCommandsArray.push(config4);
```

Comment:
I have already added two for loops now one for converters and another one for Timelocks, so i think we should avoid any further loop to avoid any confusion and maintain readability.
https://github.com/VenusProtocol/vips/pull/49/commits/0473a2acadf59f75370bf39aa959d6cd9a55476e